### ISSUE TITLE
Create Covid-19 pre-screening quiz

### ISSUE THREAD

**Issue by mputnik** [hc-sc/devops-students#9]
Quiz will utilize and showcase some of the accessibility features of WET framework for checkboxes and radio buttons. Users will be required to fill out quiz before continuing to booking: if pass→continue, if fail→err msg, return to homepage.

**Comment by jjsoong** [comment]
Changed plans
